python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -495,8 +511,13 @@ def prepareInputs(self, refList):
         tempExpRefList = []
         weightList = []
         imageScalerList = []
+        outputPsfMatchedWarpRefList = []
+
+        if not psfMatchedWarpRefList:
+            psfMatchedWarpRefList = [None for _ in range(len(refList))]
```

Comment:
Yup, this is a list comprehension ultimately. I will also change it to just `[None] * len(refList)`.